lib/beaker/host.rb

```diff
@@ -438,7 +438,7 @@ def do_scp_to source, target_path, options
           @logger.trace result.stdout
         end
       else # a directory with ignores
-        dir_source = Dir.glob("#{source}/**/*").reject do |f|
+        dir_source = Dir.glob("#{source}/**/{*,.*}").reject do |f|
```

Comment:
@pegasd this looks like a good change, but we'd like to not change the default behavior, but instead have this guarded behind an option like (for example) `:copy_hidden_files` for the method.